### DISCUSSION TITLE
Next.js image optimization 502 error

### DISCUSSION THREAD

**Discussion by Hartaithan** [vercel/community#2542] [Help]
### Summary

deployed app [https://images-test-nu.vercel.app](https://images-test-nu.vercel.app)

my `next.config.js` that's allows any image domains

```
const nextConfig = {
  reactStrictMode: true,
  images: {
    remotePatterns: [
      {
        protocol: "https",
        hostname: "**",
      },
      {
        protocol: "http",
        hostname: "**",
      },
    ],
  },
};
```

for some reason the Image component is returning a 502 error

same app deployed on netlify does not have this problem - [https://stirring-paprenjak-cd6575.netlify.app](https://stirring-paprenjak-cd6575.netlify.app)

### Additional information

Error:
```
502: BAD_GATEWAY
Code: OPTIMIZED_EXTERNAL_IMAGE_REQUEST_UNAUTHORIZED
ID: arn1::qbmxf-1684718531276-7ce26e4b21d8
```

Image component usage:

```
<Image
  className="w-auto object-contain"
  width={200}
  height={200}
  src={example.value}
  alt="test image"
/>
```

### Example

https://images-test-nu.vercel.app

**Comment by z4nr34l** [comment]
Could you bring us more details?
I consider as helpful those informations:
1. Repository Access;
2. Vercel build logs (from deployment with this issue);
3. Runtime logs from problematic request;

The more - the better for receiving quick help.

You can also try to reproduce this problem on free [CodeSandbox](https://codesandbox.io/) and post URL here when it's done.

**Reply by Hartaithan** [reply]
@z4nr34l 
1. Send you invite - [repo](https://github.com/Hartaithan/images-test)
2. Vercel logs, there's not a lot of information
<img width="1186" alt="Снимок экрана 2023-05-30 в 16 13 41" src="https://github.com/vercel/community/assets/62736094/8f888547-08c6-497a-b37c-ccf783207d42">

3. Image URL - [url](https://image.api.playstation.com/trophy/np/NPWR02322_00_00273877B6E6490ED245096C071F4DD7514629BDB1%2F6496A452C06F1E1DF7C7A8C3ED50391DE82C604C.PNG)
Optimized URL - [url](https://images-test-nu.vercel.app/_next/image?url=https%3A%2F%2Fimage.api.playstation.com%2Ftrophy%2Fnp%2FNPWR02322_00_00273877B6E6490ED245096C071F4DD7514629BDB1%2F6496A452C06F1E1DF7C7A8C3ED50391DE82C604C.PNG&w=640&q=75)
`502: BAD_GATEWAY
Code: OPTIMIZED_EXTERNAL_IMAGE_REQUEST_UNAUTHORIZED
ID: arn1::52sjn-1685431144921-350fc3f1f664`
<img width="740" alt="Снимок экрана 2023-05-30 в 16 16 36" src="https://github.com/vercel/community/assets/62736094/41c41fbc-e27a-48ae-887f-8e63a6e59c12">
